base/utils/ControlManager.ts

```diff
@@ -1,9 +1,10 @@
 import { directions, reverse_directions, action_inputs } from '../utils';
 import * as _ from "lodash";
+
 const DEFAULT_LOOP_TIME = Phaser.Timer.QUARTER >> 1;
 
 const direction_keys = ["left", "right", "up", "down"];
```

Comment:
Why not using `const direction_keys = [reverse_directions[directions.left] ...`, for instance?

Reply:
btw, wouldn't `const direction_keys =[directions.left, directions.right ...` be enough?